src/managers/config.py

```diff
@@ -415,10 +415,19 @@ def controller_kraft_client_properties(self) -> list[str]:
         """
         password = self.state.peer_cluster.controller_password
 
-        return [
+        # Strip CC properties from KRaft quorum client properties file
+        stripped_properties = list(
+            set(self.server_properties)
+            - set(KAFKA_CRUISE_CONTROL_OPTIONS.splitlines() + self.metrics_reporter_properties)
+        )
+        stripped_properties.sort()
```

Comment:
great change, this has been around for some time. Fortunately Kafka doesn't care about properties it doesn't know :)